torch/csrc/lazy/core/int128.h

```diff
@@ -0,0 +1,138 @@
+#pragma once
+
+#include <cstdint>
+#include <limits>
```

Comment:
BTW, this file I'd be pretty comfortable putting in c10/utils, it's pretty uncontroversial (unlike the hash function)